app/src/main/java/org/mozilla/fenix/compose/button/RadioButton.kt

```diff
@@ -0,0 +1,107 @@
+
+package org.mozilla.fenix.compose.button
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.RadioButtonColors
+import androidx.compose.material.RadioButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Radio Button.
+ *
+ * @param selected [Boolean] showing if the radio button is selected or not.
+ * @param modifier [Modifier] to be applied to the radio button.
+ * @param enabled [Boolean] that controls if radio button is selectable.
+ * @param onClick Callback to be invoked when radio button is clicked.
+ */
+@Composable
+fun RadioButton(
+    selected: Boolean,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    colors: RadioButtonColors = RadioButtonDefaults.colors(
+        selectedColor = FirefoxTheme.colors.iconActionSecondary,
+        unselectedColor = FirefoxTheme.colors.iconActionSecondary,
+        disabledColor = FirefoxTheme.colors.iconDisabled,
+    ),
+    onClick: () -> Unit,
+) {
+    androidx.compose.material.RadioButton(
+        selected = selected,
+        modifier = modifier,
+        enabled = enabled,
+        colors = colors,
+        onClick = onClick,
+    )
+}
+
+@Composable
+@LightDarkPreview
+private fun RadioButtonPreview() {
+    val radioOptions = listOf("One", "Two", "Three")
+    val (selectedOption, onOptionSelected) = remember { mutableStateOf(radioOptions[1]) }
+
+    FirefoxTheme {
+        Column(
+            modifier = Modifier.background(FirefoxTheme.colors.layer1),
+        ) {
+            radioOptions.forEach { text ->
+                Row(
+                    Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 15.dp)
+                        .selectable(
+                            selected = (text == selectedOption),
+                            onClick = { onOptionSelected(text) },
+                        ),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    RadioButton(
+                        selected = (text == selectedOption),
+                        onClick = { onOptionSelected(text) },
+                    )
+                    Text(
+                        text = text,
+                        modifier = Modifier.padding(start = 15.dp),
+                    )
+                }
+            }
+            Row(
```

Comment:
Let's add a new line before this to separate out the blocks for better readability

Reply:
fixed